modules/ROOT/pages/_partials/gs-adding-custom-3scale-routes.adoc

```diff
@@ -17,10 +20,9 @@ To use a custom domain, you must create a route in the OpenShift 3scale project
 * a custom domain registration
 
 .Procedure
-
 A new OpenShift route for the public has to be created.
 
-. Log in as `customer-admin`:
+. Log in as the `customer-admin`:
```

Comment:
ok, this might not be clear, there's only one user , maybe as the ... user?

Reply:
I'll remove the suggestion